docs/tasks/access-kubernetes-api/extend-api-custom-resource-definitions.md

```diff
@@ -142,11 +144,14 @@ metadata:
   resourceVersion: ""
   selfLink: ""
 ```
+{% endcapture %}
+
+{% capture discussion %}
+## Advanced topics
 
-## Advanced Topics
 ### Finalizers
-CustomResources (objects created in the schema defined by CustomResourceDefintions)
-support finalizers.  If you add a `metadata.finalizers` stanza like
+
+Custom objects support finalizers. If you add a `metadata.finalizers` stanza like:
```

Comment:
Since it's not immediately clear *what* finalizers are, would it be possible to have a one-liner definition? If that's out of scope, a link to at least https://github.com/kubernetes/community/blob/master/contributors/design-proposals/namespaces.md#finalizers would be helpful!